app/models/remote_record.rb

```diff
@@ -88,7 +92,14 @@ def attributes
   private
 
     def jsonld_request
-      @jsonld_request ||= Faraday.get("https://bibdata.princeton.edu/bibliographic/#{source_metadata_identifier}/jsonld")
+      @jsonld_request ||=
+        begin
+          request = Faraday.get("https://bibdata.princeton.edu/bibliographic/#{source_metadata_identifier}/jsonld")
+          if !self.class.alma?(source_metadata_identifier) && !request.success? && request.status.to_s == "404"
```

Comment:
Should we try in any 404 situation? given the chance of voyager IDs that fit the alma pattern?